packages/wonder-blocks-dropdown/components/multi-select.js

```diff
@@ -184,6 +225,8 @@ export default class MultiSelect extends React.Component<Props, State> {
             searchText: "",
             lastSelectedValues: [],
         };
+        // merge custom labels with the default ones
+        this.labels = {...defaultLabels, ...props.labels};
```

Comment:
I ended up creating this `labels` instance variable to merge the custom labels with the default labels defined in `constants.js`.

Reply:
Will this update if the props change later on?

Reply:
mmm It won't. I'll make the pertinent changes.